recipes/newrelic/infrastructure/debian.yml

```diff
@@ -114,6 +114,12 @@ install:
               exit 131
             fi
           fi
+        - |
+          APT_GET_STATUS=$(sudo bash -c 'apt-get update -yq > /dev/null 2>&1; echo $?')
+          if [ $APT_GET_STATUS -gt 0 ] ; then
+            echo "apt check exited with $APT_GET_STATUS code. $(sudo apt-get update -yq 2>&1 | grep "Err:")" >&2
```

Comment:
it's bugging me that we're running apt-get update twice. It shouldn't be necessary.

Reply:
in fact it will be 3 times, as one exists in the actual install step. We need to do it somehow else probably

Reply:
@brushknight a change was deployed yesterday to ignore errors due to invalid package https://github.com/newrelic/open-install-library/blob/main/recipes/newrelic/infrastructure/debian.yml#L167
Could you mirror the same `-o Acquire::Check-Valid-Until=false` to skip those potential errors?